src/windows/osd.c

```diff
@@ -349,3 +355,48 @@ ssize_t sendmsg(int sd, struct msghdr *msg, int flags)
 	sent = -1;
 	goto fn_complete;
 }
+
+/* enumerate existing addresses */
+/* in case if GetIpAddrTable is not enough, try to use
+   GetAdaptersInfo or GetAdaptersAddresses */
+void ofi_sock_get_ip_addr_table(struct slist *addr_list)
```

Comment:
nit: Let's go with just 'sock' as the prefix.  The 'ofi' prefix is being used for common, internal helper functions used by multiple providers.